app/views/books/show.html.slim

```diff
@@ -0,0 +1,47 @@
+.container
+  h1.title
+    | Book info:
+  .book-info
+    .info-item
+      span.name Title:
+      span.value = @book.title
+    .info-item
+      span.name Publishing house:
+      span.value = @book.publishing_house
+    .info-item
+      span.name Language:
+      span.value = @book.language
+    .info-item
+      span.name Year of publication:
+      span.value = @book.year_of_publication
+    .info-item
+      span.name Number of pages:
+      span.value = @book.number_of_pages
+    .info-item
+      span.name Format:
+      span.value = @book.format
+    .info-item
+      span.name Binding:
+      span.value = @book.binding
+    .info-item
+      span.name ISBN:
+      span.value = @book.isbn
+    .info-item
+      span.name Availability:
+      span.value = @book.availability
+    .info-item
+      span.name Author:
+      span.value = @book.author_ids
+    nav
+      ul.nav.navbar
+        li.nav-item = link_to 'Edit Book',
+                               edit_book_path,
+                               class: 'btn btn-outline-warning'
+        li.nav-item = link_to 'Delete Book',
+                               @book,
+                               method: :delete,
+                               data: { confirm: 'Are you sure?' },
+                               class: 'btn btn-outline-danger'
```

Comment:
edit/delete access is not offered for regular users
look at [`policies`](https://github.com/varvet/pundit#policies)